common/persistence/src/main/java/app/coronawarn/server/common/persistence/repository/CustomDiagnosisKeyRepository.java

```diff
@@ -0,0 +1,26 @@
+/*
```

Comment:
Please replace with with generated license file header by copying from other files or just running `mvn license:update-file-header` on root. (This needs to be documented still but pending https://github.com/corona-warn-app/cwa-server/issues/354)